pkg/utils/configMapUtils.go

```diff
@@ -66,17 +61,16 @@ func (expDetails *ExperimentDetails) ValidateConfigMaps(clients ClientSets) erro
 	return nil
 }
 
-func getExperimentConfigmaps(clients ClientSets, expDetails *ExperimentDetails) ([]v1alpha1.ConfigMap, error) {
+func (expDetails *ExperimentDetails) getExperimentConfigmaps(clients ClientSets) ([]v1alpha1.ConfigMap, error) {
```

Comment:
To make it more explicit, can we name this function as `getConfigMapsFromChaosExperiment`